### PR TITLE
[Review Only]Rename WSO2 references to Asgardio

### PR DESCRIPTION
**Pull request by gayashanbc**

### CODE REVIEW

File: io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package io.asgardio.android.oidc.sdk.sample;
```
Comment (darshanasbg) [inline io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java:1]: add license

Comment (gayashanbc) [reply io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java:18]: Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/ca0ccdddb9b101487384388922108627685dfb13

File: io.asgardio.android.oidc.sdk.sample/build.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 apply plugin: 'com.android.application'
 
-version = '0.0.2-SNAPSHOT'
```
Comment (darshanasbg) [inline io.asgardio.android.oidc.sdk.sample/build.gradle]: Can we inherit this from a base properties, rather defining the version again here?

File: io.asgardio.android.oidc.sdk.sample/src/main/res/raw/oidc_config.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "client_id": "tqVQ5YCvmlbdT9Z5FfK_bEnyUzca",
+  "client_id": "bdEQHU0CW8_G1HtgmGQS_hckuwsa",
   "redirect_uri": "wso2sample://oauth2",
   "authorization_scope": "openid",
-  "discovery_uri": "https://devcloud.kubesb.com/t/usertest.com/oauth2/oidcdiscovery/.well-known/openid-configuration"
+  "discovery_uri": "https://10.0.2.2:9443/oauth2/oidcdiscovery/.well-known/openid-configuration"
```
Comment (darshanasbg) [inline io.asgardio.android.oidc.sdk.sample/src/main/res/raw/oidc_config.json:5]: can we use something general like localhost?

File: io.asgardio.android.oidc.sdk.sample/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            </intent-filter>
+        </activity>
+        <activity
+            android:name="io.asgardio.android.oidc.sdk.sample.UserInfoActivity"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar" />
+        <activity
+            android:name="io.asgardio.android.oidc.sdk.sample.LoginActivity"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar" />
+    </application>
+</manifest>
```
Comment (darshanasbg) [inline io.asgardio.android.oidc.sdk.sample/src/main/AndroidManifest.xml:49]: Add a new line

File: io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java (hunk 1 above)
Comment (darshanasbg) [reply io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java:18]: Add a empty line between license and the package name

File: io.asgardio.android.oidc.sdk.sample/src/main/res/raw/oidc_config.json (hunk 3 above)
Comment (gayashanbc) [reply io.asgardio.android.oidc.sdk.sample/src/main/res/raw/oidc_config.json]: Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/84a06a42140b0b5d6153ef4b61c9ea3198933898

File: io.asgardio.android.oidc.sdk.sample/src/main/AndroidManifest.xml (hunk 4 above)
Comment (gayashanbc) [reply io.asgardio.android.oidc.sdk.sample/src/main/AndroidManifest.xml]: Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/84a06a42140b0b5d6153ef4b61c9ea3198933898

File: io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                Intent i = new Intent(SplashActivity.this, LoginActivity.class);
+                startActivity(i);
+
+                // close this activity
+                finish();
+            }
+        }, SPLASH_TIME_OUT);
+    }
+
+}
```
[the remark this answers is not in the document]
Comment (gayashanbc) [reply io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java]: Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/84a06a42140b0b5d6153ef4b61c9ea3198933898

File: io.asgardio.android.oidc.sdk.sample/build.gradle (hunk 2 above)
Comment (gayashanbc) [reply io.asgardio.android.oidc.sdk.sample/build.gradle]: Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/b54171ed692633312b0872c9da7ddf9c18bd5355

**Comment by gayashanbc** [comment]
Closing this PR as the review process is complete

**Pull request closed without merging**